src/data/mod.rs.orig

```diff
@@ -0,0 +1,92 @@
+pub use self::sin::SinSource;
```

Comment:
Please remove this file. It is part of the diff. 
```bash
git rm src/data/mod.rs.orig
```

In general: do not run `git add -A` but manually add files instead.